docs/finalproject/external-projects.md

```diff
@@ -88,7 +79,7 @@ We use the following technologies at CodeYourFuture
 
 An external Final Project should use all of these technologies into a single Full Stack package. This is important as we want our trainees to use all of the skills that they have learnt in our course.
 
-We prohibit using other core frameworks as they will fall outside our trainees areas of competency due to the time constraints that we have.
+If your tech stech is outside of the above core frameworks, send you project anyway and we will access it case-by-case.
```

Comment:
```suggestion
If your tech stack is far outside of the above core frameworks, send your project anyway and we will assess it case-by-case. 
```

This interface is a little unfriendly for long form content creation. It might be helpful to paste your changes into a Google Doc and run Grammarly on them before updating the files here.